pkg/defaults/policies/files/severity_important.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "a919ccaf-6b43-4160-ac5d-a405e1440a41",
-  "name": "Fixable Severity at least Important",
-  "description": "Alert on deployments with fixable vulnerabilities with a Severity Rating at least Important",
+  "name": "Containers with fixable CVEs and Severity at least Important",
+  "description": "Alert on containers with fixable vulnerabilities with Severity rating Important or higher. If you prefer CVSS-based policy, consider enabling 'Containers with Fixable CVEs and CVSS >= 7' instead.",
   "rationale": "Known vulnerabilities make it easier for adversaries to exploit your application. You can fix these high-severity vulnerabilities by updating to a newer version of the affected component(s).",
```

Comment:
alerts for policy violations are raised on deployment level. hence should be worded accurately - Deployment with container image with fixable CVEs and Severity at least Important